src/api/java/io/github/opencubicchunks/cubicchunks/api/worldgen/CubeGeneratorsRegistry.java

```diff
@@ -53,6 +53,9 @@
 
 public class CubeGeneratorsRegistry {
 
+    /** List of populators added by other mods to FlatCubic generator type */
+    public static final List<ICubicPopulator> customPopulatorsForFlatCubicGenerator = new ArrayList<ICubicPopulator>();
```

Comment:
Don't make this public. Do it the same way it's done for normal populators. This list should not be directly accessible to mods.